src/components/globalAlert/GlobalAlert.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  GlobalAlert,
+  GlobalAlertProps
+} from "./GlobalAlert";
```

Comment:
This repo tries to avoid relative directory paths when possible. Can you change the import path to `components/notification/GlobalAlert`?

Typescript will resolve the `components` folder to the correct directory